README.md

```diff
@@ -1,28 +1,75 @@
 # besluit-publicatie-melding-service
-This service listens for new publised
-This service listens for resource [signing:PublishedResource](http://mu.semte.ch/vocabularies/ext/signing/PublishedResource), with status `<http://mu.semte.ch/vocabularies/ext/besluit-publicatie-publish-service/status/success>`
-The service then notifies an endpoint, compliant with the [submission-api](https://lblod.github.io/pages-vendors/#/docs/submission-api).
-In practice this should be [automatic-submission-service](https://github.com/lblod/automatic-submission-service).
+This service listens for resources of type [signing:PublishedResource][publishedResource], with status `<http://mu.semte.ch/vocabularies/ext/besluit-publicatie-publish-service/status/success>`
+The service then notifies an endpoint, compliant with the [submission-api][submission-api]
+In practice this will usually be [automatic-submission-service][automatic-submission-service].
 
 ## Rest API
-For debugging:
-- `POST /submit-publication`: endpoint for [mu-delta](https://github.com/mu-semtech/delta-notifier) to trigger.
+
+- `POST /submit-publication`: endpoint for [mu-delta][mu-delta] to trigger. 
+Can also be used to manually trigger the service for debugging purposes.
+
 
 ## Params
 ```
 besluit-publicatie:
     image: lblod/besluit-publicatie-melding-service:z.y.x
     environment: # defaults params are shown here
       KEY: 'api-key-as-provided' # (required)
-      MAX_ATTEMPTS: '10'
-      PUBLISHER_URI: '"http://data.lblod.info/vendors/gelinkt-notuleren'
       SUBMISSION_ENDPOINT: 'http://automatic-submission-service/melding' # The target endpoint to submit to (required)
-      SOURCE_HOST: 'http://the.public.host.where.publication.is.availible' # (required)
+      SOURCE_HOST: 'http://the.public.host.where.publication.is.available' # (required)
+
+      PUBLISHER_URI: 'http://data.lblod.info/vendors/gelinkt-notuleren'
       DEFAULT_GRAPH: 'http://mu.semte.ch/graphs/public'
+      CACHING_CRON_PATTERN: '0 */5 * * * *'
+      PENDING_TIMEOUT_HOURS: '3'
+      MAX_ATTEMPTS: '10' 
+      PING_DB_INTERVAL: '2'
     links:
       - database:database
 ```
 
+#### KEY (required)
+
+Provided api key for submitting to the `SUBMISSION_ENDPOINT`.
+
+#### SUBMISSION_ENDPOINT (required)
+
+URI for the endpoint where the notification (aka "melding") will be submitted to.
+This endpoint must comply with the [submission-api][submission-api] specification.
+
+#### SOURCE_HOST (required)
+
+Base url where the published resource can be publicly viewed. Used for generating
```

Comment:
not sure if you mean the following, but afaik this is used to generate urls that will be submitted to the submission endpoint

Reply:
Ah that i didnt know, ill add it